rfcs/rfc7-contributor-incident-maintenance-logging.md

```diff
@@ -0,0 +1,177 @@
+
+Contributors can create incidents or maintenance via a shared form (backed by Airtable or similar) which submits through the API gateway, or directly through the API itself. Slack remains the primary interface for awareness, conversation, and history, all open and historical events appear in `#contributor-incidents` and `#contributor-maintenance`.
+
+Over time, minimal facts are anchored on the DoubleZero ledger so any party can verify contributor-reported status or build independent explorers without relying on centralized databases.
+
+# **Motivation**
+
+Operational awareness across contributors is fragmented in private systems and Slack DMs / Threads. We need a uniform, verifiable way to publish incidents (unplanned) and maintenance (planned), with searchable shared context and cross-contributor visibility.
+
+A unified gateway gives one authoritative ingress for operational events. Slack remains the collaboration surface, but the canonical record of status and lifecycle changes becomes machine-verifiable and future-proof for decentralization.
+
+This design delivers immediate value (shared visibility, consistent status logs, network-wide coordination) while maintaining a clean migration path to on-chain anchoring.
+
+# **New Terminology**
+
+| Term | Definition |
+| --- | --- |
+| **Incident** | Unplanned service-impacting event with enumerated severity and status. |
+| **Maintenance** | Planned, time-bounded activity that may affect availability. Auto-closes after `end_at`. |
+| **Proof of Contributor** | Caller authority via per-contributor API key (MVP); later, signed requests and on-chain registry. |
+| **Gateway** | Authenticated, versioned ingress that validates identity, enforces schema, persists events, and emits notifications. |
+
+# **Alternatives Considered**
+
+- **Do nothing:** Keeps fragmented visibility and manual coordination.
+- **Point-to-point bridges:** N² maintenance trap across diverse contributor systems.
+- **Mandate one product:** Simplifies normalization but undermines autonomy.
+- **Go fully on-chain day one:** Slows adoption and tooling readiness.
+
+A **gateway-first** design achieves fast adoption and a clear path to progressive decentralization.
+
+# **Detailed Design**
+
+## Architecture
+
+- **Ingress**: Versioned REST API and web form (writes through the API).
+- **Identity**: API key bound to a Contributor `pubkey` (service key).
+- **Storage**: Airtable (MVP) →  on-chain anchors.
+- **Notifications**: Slack channel for new records, status changes (MVP).
+
+## Cross-Contributor Tickets
+
+- Any contributor can open a ticket for any device or link. (e.g., observed issues on another contributors device/link).
+- Contributors can mention each other by using the slack `@<conbributor>`.
```

Comment:
Is there a Slack team/group per contributor or how do they know who's who or who the contact @ is?